### PR TITLE
Language filter testing

### PR DESCRIPTION
**Pull request by benhayehudi**
## Description

Added testing for `LanguageFilter`, specifically:

* It does nothing with non-matching text
* It raises an error if the input is only whitespace
* It correctly transforms matching input
* It does nothing if the input is correct except for the formatting

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: spec/filters/language_filter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    expected_output = 'some text'
+
+    expect(described_class.call(input)).to eql(expected_output)
+  end
+
+  it 'raises an error if input is whitespace' do
+    input = ' '
+
+    expected_output = ' ' # rubocop:disable UselessAssignment
```
Comment (mheap) [inline spec/filters/language_filter_spec.rb]: Rubocop was correct, this is a useless assignment. We can delete this line

File: spec/filters/language_filter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    expect { described_class(input) }.to raise_error(ArgumentError)
```
Comment (mheap) [inline spec/filters/language_filter_spec.rb]: Should this be `described_class.call()` ?

File: spec/filters/language_filter_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  it 'wraps matching input in a span tag and assigns "lang" to the tag' do
+    input = "[נצמו](lang: 'il')"
+
+    expected_output = "<span lang='il'>נצמו</span>"
+
+    expect(described_class.call(input)).to eql(expected_output)
+  end
+
+  it 'does nothing if the text is in correct format but it is missing the correct formatting' do
```
Comment (mheap) [inline spec/filters/language_filter_spec.rb]: How can the text be in the correct format but missing the correct formatting?

Comment (benhayehudi) [reply spec/filters/language_filter_spec.rb]: That made sense when I wrote it! 😉 I've updated it now to be more accurate.

File: spec/filters/language_filter_spec.rb (hunk 2 above)
Comment (benhayehudi) [reply spec/filters/language_filter_spec.rb]: Yes, that is correct, and when doing so it no longer raises an error, but returns the whitespace input. I've modified the test to more accurately describe the current behavior.

File: spec/filters/language_filter_spec.rb (hunk 1 above)
Comment (benhayehudi) [reply spec/filters/language_filter_spec.rb]: This has been resolved as a result of the test rewrite.

**Pull request merged**